PatternPal/PatternPal.Core/Checks/ModifierCheck.cs

```diff
@@ -1,7 +1,12 @@
 ﻿namespace PatternPal.Core.Checks;
 
+/// <summary>
+/// Checks for the modifiers of an entity. Depending on the <see cref="List{T}"/> of <see cref="IModified"/> provided.
+/// It should match with the modifiers of the current <see cref="INode"/> being checked.
+/// </summary>
 internal class ModifierCheck : CheckBase
 {
+    // all modifiers the node should have
     private readonly List< IModifier > _modifiers;
 
     public ModifierCheck(Priority priority,
```

Comment:
Documentation comments should be added to all top level elements like classes, methods and properties.